tests/functions_tests/test_dropout.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import testing
+from chainer.testing import attr
+
+
+if cuda.available:
+    cuda.init()
+
+
+class TestDropout(unittest.TestCase):
+
+    def setUp(self):
+        self.x = numpy.random.uniform(-1, 1, (2, 3)).astype(numpy.float32)
+
+    def check_type_forward(self, x_data):
+        x = chainer.Variable(x_data)
+        try:
+            functions.dropout(x)
+        except Exception:
```

Comment:
Is it required? When an exception occurred in unit test, the test will fail automatically.